src/feature/dirparse/microdesc_parse.c

```diff
@@ -92,6 +92,9 @@ find_start_of_next_microdesc(const char *s, const char *eos)
 #undef NEXT_LINE
 }
 
+#define POLICY_IS_REJECT_STAR(policy) (policy == NULL || (!policy && \
```

Comment:
This logic is wrong:
```suggestion
#define POLICY_IS_REJECT_STAR(policy) (!policy \
```

And please remove the extra brackets.